src/openrct2/actions/RideSetAppearanceAction.h

```diff
@@ -29,7 +29,7 @@ DEFINE_GAME_ACTION(RideSetAppearanceAction, GameCommand::SetRideAppearance, Game
 private:
     NetworkRideId_t _rideIndex{ RIDE_ID_NULL };
     RideSetAppearanceType _type{};
-    uint8_t _value{};
+    uint16_t _value{};
```

Comment:
Is this ObjectEntryIndex? Also the type was not changed in the constructor.

Reply:
_Sometimes_ it’s ObjectEntryIndex. `RideSetAppearanceAction` is used for several different settings.

Reply:
I see, either way you should probably changed uint8 to uint16 of the constructor parameter.

Reply:
That was an oversight. I have now adjusted them.